### ISSUE TITLE
Consistent coding conventions?

### ISSUE THREAD

**Issue by ThereWasADream** [Jessecar96/SteamBot#781]
While working on a pull request for Bot.cs, I noticed there didn't seem to be a consistent coding convention. Furthermore, Contributing.md (https://github.com/Jessecar96/SteamBot/blob/master/CONTRIBUTING.md) doesn't include guidelines for more than indentation and brackets.

I was under the impression you all use Microsoft's coding conventions, but when I reformatted under MS Visual Studio's rules, there were more changes than I would have expected. Since I don't want to mess up my first pull request, I thought I'd ask if this would be expected or not.

See differences here: https://github.com/Jessecar96/SteamBot/compare/master...ThereWasADream:Pull-Requests

(This includes the other changes, not just the formatting changes)

If everyone's OK with the format changes, I'll just go ahead with the pull request. If not, what rules should I be using?

**Comment by BlueRaja** [comment]
I agree, we should reformat the files someday to be more consistent.  The reason I never did is that it would break the existing PR's.

**Comment by ThereWasADream** [comment]
In that case, if there aren't any rules in place already, I think we should
just use MS Visual studio's rules. They seem to be as close to what
Steambot already uses as possible.
